cmd/operator-sdk/test/local.go

```diff
@@ -117,7 +117,12 @@ func testLocalGoFunc(cmd *cobra.Command, args []string) error {
 	}
 
 	if tlConfig.upLocal && tlConfig.namespace == "" {
-		return fmt.Errorf("must specify a namespace to run in when -up-local flag is set")
+		// Allows namespace == "" with --up-local in order to support cluster-scoped operators.
+		// Then, it will allow users to execute the command operator-sdk test local --up-local with the flag --namespace="" (allowed only with --no-setup).
+		// More Info: https://github.com/operator-framework/operator-sdk/blob/master/doc/operator-scope.md#cluster-scoped-operator-usage
+		if !(cmd.Flags().Changed("namespace") && tlConfig.noSetup) {
+			return fmt.Errorf("must specify a namespace to run in when -up-local flag is set without setting -no-setup flag")
+		}
 	}
 
```

Comment:
IMHO: By reading the code should be easier to check what flag or value was not satisfied. So, maybe the following could clarifies. 

```suggestion
if tlConfig.upLocal && tlConfig.namespace == "" && !(cmd.Flags().Changed("namespace") && tlConfig.noSetup) {
       return fmt.Errorf("must specify a namespace to run in when -up-local flag or use the -no-setup flag to disable the test resource creation ")
}
```

Also, why you add the `tlConfig.noSetup`. Could not it be simplified by: 


```suggestion
if tlConfig.upLocal && tlConfig.namespace == "" && !cmd.Flags().Changed("namespace") {
       return fmt.Errorf('must specify a namespace to run in when -up-local flag is set. Note that to allow WATH all you can use -namespace="" ')
}
```